src/Fantomas.Core/SyntaxOak.fs

```diff
@@ -1461,11 +1461,89 @@ type ExprTryFinallyNode(tryNode: SingleTextNode, tryExpr: Expr, finallyNode: Sin
     member x.Finally = finallyNode
     member x.FinallyExpr = finallyExpr
 
-type ExprIfThenNode(ifNode: MultipleTextsNode, ifExpr: Expr, thenNode: SingleTextNode, thenExpr: Expr, range) =
+type ElseIfNode(mElse: range, mIf: range, condition: Node, range) as elseIfNode =
+    let nodesBefore = Queue<TriviaNode>(0)
+    let nodesAfter = Queue<TriviaNode>(0)
+    let mutable lastNodeAfterIsLineCommentAfterSource = false
+
+    let elseNode =
+        { new Node with
+            member _.ContentBefore: TriviaNode seq = Seq.empty
+            member x.HasContentBefore: bool = false
+            member _.ContentAfter: TriviaNode seq = Seq.empty
+            member x.HasContentAfter: bool = false
+            member _.Range = mElse
+
+            member _.AddBefore(triviaNode: TriviaNode) =
+                (elseIfNode :> Node).AddBefore triviaNode
+
+            member _.AddAfter(triviaNode: TriviaNode) =
+                (elseIfNode :> Node).AddAfter triviaNode
+
+            member _.Children = Array.empty }
+
+    let ifNode =
+        { new Node with
+            member _.ContentBefore: TriviaNode seq = Seq.empty
+            member x.HasContentBefore: bool = false
```

Comment:
Mmh, I might be missing something obvious, but why the switch from `_` to `x` and vice versa in the members?

Reply:
Because this code started out as a copy of `NodeBase` and I changed members afterwards.
We perhaps also should settle on some code convention for this. 
When to use `_`. When `x` vs `this` vs `self`,..

Reply:
Yes, good idea. But for the moment, I'm totally fine with just being consistent in the scope of a definition.